### PR TITLE
Update to spdlog 1.8.2

### PR DESCRIPTION
**Pull request by jeandet**

### CODE REVIEW

**Comment by jpakkane** [comment]
Retargeted, please squash to remove extra middle commit.

**Comment by jeandet** [comment]
Yep, done.

File: upstream.wrap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = spdlog-1.8.2
+source_url = https://github.com/gabime/spdlog/archive/v1.8.2.tar.gz
+source_filename = v1.8.2.tar.gz
+source_hash = e20e6bd8f57e866eaf25a5417f0a38a116e537f1a77ac7b5409ca2b180cec0d5
```
Comment (jpakkane) [inline upstream.wrap:6]: This is missing a `[provide]` section, which has been recently made mandatory.

Comment (eli-schwartz) [reply upstream.wrap:6]: There is one in the 1.8.1 wrap, which can be copied over...

File: upstream.wrap [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = spdlog-1.8.2
```
Comment (eli-schwartz) [inline upstream.wrap:2]:
```suggestion
directory = spdlog-1.8.2

```

Could you add a blank line here, for readability?

**Pull request merged**